src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.classes`. The `.stars` class is directly applied to a `ul` element, which should be styled by its class instead of using the tag name as a selector.